src/directives/layertree.js

```diff
@@ -4,15 +4,25 @@ goog.provide('ngeo.layertreeDirective');
 goog.require('ngeo');
 
 
+/**
+ * Unique template path for the layertree;
+ * @type {string|undefined}
+ */
+ngeo.layertreeTemplateUrlValue = undefined;
```

Comment:
Whitout that, you can have a template path defined twice: 
 - in the definition of the ngeo-layertree (for instance in the index.html)
 - in the ngeo-layertree-template itself

In case of GMF, the path is defined once in the gmf layertree directive and once in the gmf layertree template (or if not, each leaf will take ``ngeo.baseTemplateUrl + '/layertree.html'`` , and that can be another path that in the root element).

Force to have only one template is a good thing I think. But I think that define variables like that is not a clean way do it. Someone has any advice ?